test/beer-router-test.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+require('dotenv').load();
+const superagent = require('superagent');
+const expect = require('chai').expect;
+const Beer = require('../model/beer.js');
+const baseUrl = `http://localhost:${process.env.PORT || 3000}`;
+require('../server.js');
+
+//testing POST
+describe('testing beer router', function() {
+  describe('testing POST /api/beers', function() {
+    after((done) => {
+      Beer.deleteById(this.tempbeer.id)
+      .then(() => done())
+      .catch(done);
+    });
+
+    it('should create a beer with 200 status', done => {
+      superagent.post(`${baseUrl}/api/beers`)
+        .send({
+          name: 'Double Dead Guy',
+          type: 'IPA',
+          strength: '7.9',
+        })
+        .then(res => {
+          expect(res.status).to.equal(200);
+          expect(res.body.name).to.equal('Double Dead Guy');
+          expect(res.body.type).to.equal('IPA');
+          expect(res.body.strength).to.equal('7.9');
+          expect(Boolean(res.body.id)).to.equal(true);
+          expect(Boolean(res.body.created)).to.equal(true);
+          this.tempbeer = res.body;
```

Comment:
When you have a lot of properties to test for, writing all these expect statements can be a pain. I refactored these expect statements to the following code:

```
for(let key in res.body){
          if (key === 'created'){
            res.body[key] = new Date(res.body.created.toString());
          }
          expect(res.body[key].toString()).to.equal(this.tempBeer[key].toString());
        }
```